mmv1/templates/terraform/examples/datastream_stream_mysql_gtid.tf.erb

```diff
@@ -0,0 +1,103 @@
+resource "google_sql_database_instance" "instance" {
+    name                = "<%= ctx[:vars]['mysql_name'] %>"
```

Comment:
Also ensure to set prevent destroy to false. Unless this is intended to be a docs only config.

Reply:
Done.